src/OpenTelemetry.Instrumentation.Dependencies/Implementation/HttpWebRequestActivitySource.net461.cs

```diff
@@ -32,21 +33,21 @@ namespace OpenTelemetry.Instrumentation.Dependencies.Implementation
     /// Created from the System.Diagnostics.DiagnosticSource.HttpHandlerDiagnosticListener class which has some bugs and feature gaps.
     /// See https://github.com/dotnet/runtime/pull/33732 for details.
     /// </remarks>
-    internal sealed class HttpWebRequestDiagnosticSource : DiagnosticListener
+    internal sealed class HttpWebRequestActivitySource
     {
-        internal const string DiagnosticListenerName = "HttpWebRequestDiagnosticListener";
-        internal const string ActivityName = DiagnosticListenerName + ".HttpRequestOut";
-        internal const string RequestStartName = ActivityName + ".Start";
-        internal const string RequestStopName = ActivityName + ".Stop";
-        internal const string RequestExceptionName = ActivityName + ".Exception";
+        internal const string ActivitySourceName = "HttpWebRequest";
+        internal const string ActivityName = ActivitySourceName + ".HttpRequestOut";
 
-        internal static readonly HttpWebRequestDiagnosticSource Instance = new HttpWebRequestDiagnosticSource();
+        internal static readonly HttpWebRequestActivitySource Instance = new HttpWebRequestActivitySource();
 
-        private const string InitializationFailed = DiagnosticListenerName + ".InitializationFailed";
+        private const string InitializationFailedActivityName = ActivitySourceName + ".InitializationFailed";
         private const string CorrelationContextHeaderName = "Correlation-Context";
         private const string TraceParentHeaderName = "traceparent";
         private const string TraceStateHeaderName = "tracestate";
 
+        private static readonly Version Version = typeof(HttpWebRequestActivitySource).Assembly.GetName().Version;
+        private static readonly ActivitySource ActivitySource = new ActivitySource(ActivitySourceName, Version.ToString());
```

Comment:
can you rename field to something other than just ActivitySource. like WebRequestActivitySource?

Reply:
Sure